contracts/libraries/LibBonds.sol

```diff
@@ -77,8 +77,11 @@ library LibBonds {
         uint256 diggingFeesDue = archaeologist.diggingFeePerSecond *
             (sarcophagus.resurrectionTime - sarcophagus.previousRewrapTime);
 
-        decreaseFreeBond(archaeologist.archAddress, diggingFeesDue);
-        s.archaeologistProfiles[archaeologist.archAddress].cursedBond += diggingFeesDue;
+        // Use cursed bond percentage to determine how much bond to lock up
+        uint256 bondToCurse = (diggingFeesDue * s.cursedBondPercentage) / 100;
```

Comment:
I'm sure this was the plan already, but just for reference so it doesn't somehow slip through the cracks, the returning `calculateCursedBond` function can be used for this (and other) cursedBond determining logic.

Reply:
@DarksightKellar -- I need to run some gas comparisons. If it's cheaper to run it inline like this, then I might remove the `calculateCursedBond` all together

Reply:
Yea now that you mention it, I have asked this question before (function calls vs inline statements), and the gas comparison I did showed inline was marginally but not-negligibly cheaper. I'll let you run the comparisons on *this* code to confirm, in which case removing `calculateCursedBond` might be the play